src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall().toString());
+        System.out.println(lottery.getRandomBall().toString());
+        System.out.println(lottery.getRandomBall().toString());
```

Comment:
Instead of calling `lottery.getRandomBall().toString()` multiple times, consider storing the result in a variable if the `toString()` method is not expensive or if the object is not meant to be unique on each call. If the `Ball` objects are meant to be unique, then this approach is fine.